__mocks__/addAndRemove.js

```diff
@@ -24,26 +24,13 @@ class UserTask {
     } if (!checkItem) {
       toDoList.push(this);
       localStorage.setItem('TaskToday', JSON.stringify(toDoList));
-      const testing = JSON.parse(localStorage.getItem('TaskToday'));
-      document.body.innerHTML
-      += `<div > 
-        <span id="username" /> 
-        <img id="imtrash${testing.length}"
-        <button class="imgRemove" /> 
-        <input type="checkbox" class="check" />
-      </div>`;
-      return testing.length;
-      // location.reload();
+      return toDoList.length;
     }
     const ArrayStored = localStorage.getItem('TaskToday');
     const ArrayStoredParse = JSON.parse(ArrayStored);
```

Comment:
It would be better if you declared and parsed the value at once, this would reduce the lines of code following the KISS and YAGNI principles